phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    excludes_analyse:
+        - */public/index.php
+        - */scripts/*
+    paths:
```

Comment:
What do you think about keeping parameters sorted by name here?

This way it will be more obvious where to add new parameters.

Reply:
Sure, I can do that.